djautotask/admin.py

```diff
@@ -132,3 +132,14 @@ class AccountAdmin(admin.ModelAdmin):
 class ProjectAdmin(admin.ModelAdmin):
     list_display = ('id', 'name', 'number')
     search_fields = ('id', 'name', 'number')
+
+
+@admin.register(models.Task)
+class TaskAdmin(admin.ModelAdmin):
+    list_display = ('id', 'title', 'number')
```

Comment:
For Connectwise I found it useful to have the status available for a ticket-like object to be in the list view, so it is available at a glance, or filtering. Please add that here. Actually on second thought maybe as a search field too.

Reply:
Yeah, very useful. I've added status to the list display, search field and added it as a list_filter.

I also found another issue while cleaning these up. I forgot that order is important for Method Resolution Order. So for `TaskSynchronizer(Synchronizer, QueryConditionMixin,)` the mixin override of _get_query_conditions() was not actually running and wasn't filtering out `Complete` tasks. I've reversed the order to `TaskSynchronizer(QueryConditionMixin, Synchronizer` now so complete tasks and tickets are not synced again.